tasks/pyproject.toml

```diff
@@ -23,6 +23,7 @@ python = "^3.9"
 robocorp-log = ">=2.4,<3"
 psutil = "^5.9"
 docstring_parser_fork = "^0.0.5"
+pydantic = "^2.5.2"
```

Comment:
Humm, at the tasks level I don't think we should use pydantic, just use a "plain" dataclass for the use-case if needed.

Reply:
sure, will revert.

I just prefer to use validation models in cases like this (e.g. zod in node land). We get
1. automatic data validation with nice errors.
2. typings out of the box

Reply:
As a note, when using dataclasses:
- type-validation does work already with mypy (so, static analysis is ok)
- typings are out of the box too

It doesn't do the runtime type-checking though. That's a tradeoff (so, it's faster), but on (the few) cases where you are having user input you need to validate manually.